moped-editor/src/queries/tableLookups.js

```diff
@@ -14,7 +14,7 @@ export const TABLE_LOOKUPS_QUERY = gql`
         subphase_order
       }
     }
-    moped_milestones {
+    moped_milestones(order_by: { milestone_id: asc }) {
```

Comment:
without this it seems to be ordering by when the milestone record was last updated

Reply:
nice catch! 🙌